docs/content/manual/manual.yml

```diff
@@ -854,6 +854,21 @@ sections:
             input: '[1,0,-1]'
             output: ['1', '-1']
 
+      - title: "`abs`"
+        body: |
+
+          The builtin function `abs` is defined naively as: `if . < 0 then - . else . end`.
+
+          For numeric input, this is the absolute value.  See the
+          section on the identity filter for the implications of this
+          definition for numeric input.
+
+          To compute the absolute value of a number as a floating point number, you may wish use `fabs`.
+
+        examples:
+          - program: 'map(abs)'
+            input: '[-10, -1.1, -1e-1, 1000000000000000002, -1000000000000000002]'
+            output: ['[10,1.1,1e-1,10000000000000002,1e+18]']
```

Comment:
```suggestion
            output: ['[10,1.1,0.1,1000000000000000002,1e+18]']
```

Reply:
As the corner case, `0` should be listed here.

Reply:
I also think `-1000000000000000002` shouldn't be here. The behavior is not guaranteed.

Reply:
* Added tests for 0 and -0
* Removed the contingent test in manual.yml
* Used "NOT prescriptive" annotation in jq.test